### PR TITLE
1/28 change

### PR DESCRIPTION
**Pull request by qaz6750**
#### MSM8998
* Add Mass Storage function support for sagit https://github.com/edk2-porting/edk2-msm/commit/e737c310224bbeb4f554cfe60abbd987d4fb71d4
* Enable Buttons in UEFI for sagit https://github.com/edk2-porting/edk2-msm/commit/e737c310224bbeb4f554cfe60abbd987d4fb71d4
#### SDM660
* Add initial support for whyred https://github.com/edk2-porting/edk2-msm/commit/ca22d29b79f1849f374c778ef7e27dcfef92e550
#### Submodule
* Upgrade EFI_ Binaries submodule https://github.com/edk2-porting/edk2-msm/commit/4f4250e3a0cb231dde167d345d07ea6997fb7dde
### Checklist

* [Y] Have you tested the change you are submitting?
* [Y] Is the commits history nice and clean?

### CODE REVIEW

**Review by TheMorc — APPROVED** [review]

File: Platform/Xiaomi/sdm660/whyred.dsc [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+[BuildOptions.common]
+  GCC:*_*_AARCH64_CC_FLAGS = -DENABLE_SIMPLE_INIT
+
+[PcdsFixedAtBuild.common]
+  gQcomTokenSpaceGuid.PcdMipiFrameBufferWidth|1080
+  gQcomTokenSpaceGuid.PcdMipiFrameBufferHeight|2160
+
+  gRenegadePkgTokenSpaceGuid.PcdDeviceVendor|"Redmi"
+  gRenegadePkgTokenSpaceGuid.PcdDeviceProduct|"5"
+  gRenegadePkgTokenSpaceGuid.PcdDeviceCodeName|"whyred"
```
Comment (BigfootACA) [inline Platform/Xiaomi/sdm660/whyred.dsc:24]: using simple-init but missing PcdGuiDefaultDPI?

Comment (qaz6750) [reply Platform/Xiaomi/sdm660/whyred.dsc:24]: Oh, I think I ignored it. I will solve it immediately

File: Platform/Xiaomi/sdm660/whyred.dsc [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+[BuildOptions.common]
+  GCC:*_*_AARCH64_CC_FLAGS = -DENABLE_SIMPLE_INIT
+
+[PcdsFixedAtBuild.common]
+  gQcomTokenSpaceGuid.PcdMipiFrameBufferWidth|1080
+  gQcomTokenSpaceGuid.PcdMipiFrameBufferHeight|2160
+
+  gRenegadePkgTokenSpaceGuid.PcdDeviceVendor|"Redmi"
+  gRenegadePkgTokenSpaceGuid.PcdDeviceProduct|"5"
```
Comment (BigfootACA) [inline Platform/Xiaomi/sdm660/whyred.dsc:23]: Redmi 5 or Redmi Note 5?

Comment (qaz6750) [reply Platform/Xiaomi/sdm660/whyred.dsc]: https://github.com/edk2-porting/edk2-msm/pull/247/commits/cc43d377638a5c6a8a0b1f5a912f679c41298b45 Improved these errors

**Review by BigfootACA — APPROVED** [review]

**Pull request merged**